setup.py

```diff
@@ -1,81 +1,33 @@
 """PyRACF setup/build configuration."""
```

Comment:
Since this only really describes the build process for the python extension now.
```python
"""Build python extension for pyRACF."""
```

Reply:
Agree, changed